client/src/components/AutocompleteInput/errors.scss

```diff
@@ -0,0 +1,11 @@
+.c-wagtailautocomplete__error-message {
+  color: $red;
+}
+
+
+// .help-block {
+//  clear: both;
+//  color: var(--w-color-grey-600);
+//  margin: 1em 0;
+//  padding: 1em;
+// }
```

Comment:
@chigby I think we can remove this comments?

Reply:
oops